__tests__/app.test.js

```diff
@@ -163,6 +163,58 @@ describe('POST /api/articles/:article_id/comments', () => {
   });
 });
 
+describe('PATCH /api/articles/:article_id', () => {
+  test('PATCH:201 updates the vote count of a specified article', () => {
+    const newVote = {
+      inc_votes: 2
+    };
+    return request(app)
+      .patch('/api/articles/1').send(newVote)
+      .expect(201)
+      .then((response) => {
+        expect(response.body.result.votes).toBe(102);
```

Comment:
Some destructuring would neaten this up! 

Also I would check that the article id is 1!